packages/core/package.json

```diff
@@ -80,10 +80,9 @@
     "src"
   ],
   "scripts": {
-    "prepare": "yarn run clean && yarn run build",
-    "clean": "theiaext clean",
+    "lint": "theiaext lint",
     "build": "theiaext build",
```

Comment:
please keep watch scripts that watchin individual packages is possible with `npx run watch @theia/core`

if watching individual packages should be done differently please update docs

btw we have to review all docs to check whether they mention any removed scripts

Reply:
I think this is a mistake when I was mass-editing, will add those scripts back.